vendor/README.md

```diff
@@ -0,0 +1,31 @@
+# Third party packages
+
+This directory contatins packages from third-party vendors reused in Astar Network.
+
+### Why not fork?
+
+As a way to protect code from **unexpected changes** and **release from third-party project
+internal dependencies** a dedicated folder used instead of repository fork.
+
+In case of internal packages the porter persion will have no intention to just scroll down
+repository commit history and adjust final result to the repo. Copy/paste the code risk is
```

Comment:
The _porter person_ is the uplifter? I find the sentence quite confusing, sorry.

Perhaps this can be rephrased to something like:
> When porting changes from the external _vendor_ projects into `astar-frame` repo, changes will be visible as part of the difference introduced by PR. This ensures we don't introduce any unintentional changes without being aware of them.

(this is just how I understood what you're saying here)

Reply:
Thank you, used your interpretation.